Blaise.Cati.Tests.Behaviour/app.config

```diff
@@ -1,60 +1,61 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
-  <configSections>
-    <section name="specFlow" type="TechTalk.SpecFlow.Configuration.ConfigurationSectionHandler, TechTalk.SpecFlow" />
-  </configSections>
-  <specFlow>
-    <stepAssemblies>
-      <stepAssembly assembly="Blaise.Tests.Behaviour" />
-      <stepAssembly assembly="Blaise.Cati.Tests.Behaviour" />
-    </stepAssemblies>
-  </specFlow>
-  <appSettings>
-    <add key="InstrumentPath" value="#{InstrumentPath}#" />
-    <add key="ServerParkName" value="#{ServerParkName}#" />
-    <add key="InstrumentName" value="#{InstrumentName}#" />
+	<configSections>
+		<section name="specFlow" type="TechTalk.SpecFlow.Configuration.ConfigurationSectionHandler, TechTalk.SpecFlow" />
+	</configSections>
+	<specFlow>
+		<stepAssemblies>
+			<stepAssembly assembly="Blaise.Tests.Behaviour" />
+			<stepAssembly assembly="Blaise.Cati.Tests.Behaviour" />
+		</stepAssemblies>
+	</specFlow>
+	<appSettings>
+		<add key="UninstallSurveyTimeOutInSeconds" value="5" />
```

Comment:
Did you intend to commit a hard-coded variable?

Reply:
just changing it